teradatalabs/cdh5-hive-kerberized/files/supervisord.d/hive-server2.conf

```diff
@@ -0,0 +1,8 @@
+[program:hive-server2]
```

Comment:
I think the `hive.security.authorization.manager` property is set both in command line and in hive-site.xml